Formula/libtiff.rb

```diff
@@ -15,8 +15,10 @@ class Libtiff < Formula
 
   option :universal
   option :cxx11
+  option "with-lzma", "Include support for LZMA compression"
```

Comment:
I don't think very many users actually need it, and it adds a dep. So I'd argue against making it default.

LZMA doesn't seem to be supported by many viewers, either. (e.g. OS X Preview doesn't support it, and nor does Adobe Photoshop). 

However I happened to need it for some testing, and it's pretty easy to have it there just in case.